streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -260,15 +259,14 @@ public void onPartitionsAssigned(final Collection<TopicPartition> assignment) {
                 taskManager.createTasks(assignment);
                 streamThread.refreshMetadataState();
             } catch (final Throwable t) {
-                log.error("{} Error caught during partition assignment, " +
-                        "will abort the current process and re-throw at the end of rebalance: {}", logPrefix, t.getMessage());
+                log.error("Error caught during partition assignment, " +
+                        "will abort the current process and re-throw at the end of rebalance: {}", t.getMessage());
                 streamThread.setRebalanceException(t);
             } finally {
-                log.info("{} partition assignment took {} ms.\n" +
+                log.info("partition assignment took {} ms.\n" +
```

Comment:
same here captialize

Reply:
`completed` is in lower case because before this we have applied a `logPrefix` using `LogContext`